src/components/Attachments/AttachmentsActions.js

```diff
@@ -0,0 +1,49 @@
+import { createAction } from 'redux-actions';
+
+export const addUploadingFiles = createAction(
+  'ADD_UPLOADING_FILES',
+  (files, uploadFolderId) => ({
+    files,
+    uploadFolderId,
+  }),
+);
+
+export const fileUploadInitialized = createAction(
+  'FILE_UPLOAD_INITIALIZED',
+  (file, uploadFolderId) => ({ file, uploadFolderId }),
+);
+
+export const fileUploadGetLocation = createAction(
+  'FILE_UPLOAD_GET_LOCATION',
+  (file, uploadFolderId) => ({ file, uploadFolderId }),
+);
+
+export const fileUploadingStarted = createAction(
+  'FILE_UPLOADING_STARTED',
+  (file, uploadFolderId) => ({
+    file,
+    uploadFolderId,
+  }),
+);
+
+export const fileUploadingProgressUpdated = createAction(
+  'FILE_UPLOADING_PROGRESSS_UPDATED',
+  (file, uploadFolderId, progress) => ({ file, uploadFolderId, progress }),
+);
+
+export const fileUploadingSuccess = createAction(
+  'FILE_UPLOADING_SUCCESS',
```

Comment:
All the actions we have in this project end with `_ACTION`.
Not sure why Kolia did it this way. Is it fine that we will now have also actions without `_ACTION` in the end? :)

Reply:
Didn't noticed it.

![image](https://user-images.githubusercontent.com/20638114/73413158-81d6d080-4313-11ea-8cb1-e09f2d70ccfe.png)

In my opinion, it's unnecessary to add `_ACTION` suffix for action types. We will have an uncomfortable developer experience with it - `SELECTED_NOTE_CHILDREN_FETCHED_ACTION` too long and hard to read. Imagine a lot of action types with 4 words in the name. Too high price for the incomprehensible goals.

I can guess that `_ACTION` was added to mark constants for action types. It's not necessary when we are using `redux-action` or another way for action creators.

In my opinion, we can keep everything as-is for now. And decide one way for further development. I suggest to use `createAction` / `createActions` functions for making action creators and not use `_ACTION` prefix.